config/cesm/machines/config_machines.xml

```diff
@@ -433,6 +433,63 @@ This allows using a different mpirun command to launch unit tests
     </resource_limits>
   </machine>
 
+  <machine MACH="coeus">
+    <DESC>
+      Portland State University Coeus Cluster CentOS 7
+    </DESC>
+    <NODENAME_REGEX>(login[1,2].cluster|compute[0-9]*.cluster)</NODENAME_REGEX>
+    <OS>LINUX</OS>
+    <COMPILERS>gnu</COMPILERS>
+    <MPILIBS>mvapich2</MPILIBS>
+    <PROJECT>none</PROJECT>
+    <CIME_OUTPUT_ROOT>$ENV{CESMDATAROOT}/$USER</CIME_OUTPUT_ROOT>
+    <DIN_LOC_ROOT>$ENV{CESMDATAROOT}/inputdata</DIN_LOC_ROOT>
+    <DOUT_S_ROOT>$ENV{CESMDATAROOT}/archive/$CASE</DOUT_S_ROOT>
```

Comment:
Usually you will want the DOUT_S_ROOT to be $USER specific like
$CIME_OUTPUT_ROOT/archive/$CASE

Reply:
That make sense I'll fix this now